afids_utils/io.py

```diff
@@ -0,0 +1,85 @@
+"""Methods for loading and saving nifti files"""
+from __future__ import annotations
+
+import csv
+from os import PathLike
+
+import nibabel as nib
+import numpy as np
+import pandas as pd
+from numpy.typing import NDArray
+
+from afids_utils.exceptions import InvalidFiducialNumberError
+
+AFIDS_FIELDNAMES = [
```

Comment:
Maybe rename this to `FCSV_FILENAMES`, because it's specific to FCSV files and I don't think we add anything from what slicer produces

Reply:
Just a thought and not part of this PR, but would also be worth to implement reading slicer json files (which is supported for the validator)?

Reply:
Yes, I think there's a decent amount of functionality (including handling JSONs) in the validator that should come over to this repo instead